server.js

```diff
@@ -60,16 +77,21 @@ app.all('*', function (req, res, next) {
 
 
 app.get('/authenticate/:code', function(req, res) {
-  console.log('authenticating code:' + req.params.code);
+  log('authenticating code:', req.params.code, true);
   authenticate(req.params.code, function(err, token) {
-    var result = err || !token ? {"error": "bad_code"} : { "token": token };
-    console.log(result);
+    if ( err || !token ) {
+      result = {"error": "bad_code"};
```

Comment:
Replace hard-coded `”error”` and `”bad_code”` strings with named constants to make it easier to change the values in the future, in case it is ever necessary to do so.